### DISCUSSION TITLE
Letsencrypt certificate renewal

### DISCUSSION THREAD

**Discussion by bmingles** [pocketbase/pocketbase#5590] [Q&A] [answered]
When using the built in ssl support, how does pocketbase handle renewal of letsencrypt certificates?

**Comment by ganigeorgiev** [comment, marked as answer]
Renewal is done automatically by the Go std server (we use the default settings so by default it should be [30 days before expiration](https://cs.opensource.google/go/x/crypto/+/refs/tags/v0.27.0:acme/autocert/autocert.go;l=134)).

If you want to change the cert manager configuration you can attach to the [`OnBeforeServe` hook](https://pocketbase.io/docs/go-event-hooks/#onbeforeserve) and modify the `e.CertManager` field.

**Reply by bmingles** [reply]
Great thanks!
